<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialAssisWidget</class>
 <widget class="QWidget" name="SerialAssisWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="maximumSize">
        <size>
         <width>550</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>串口设置</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="verticalSpacing">
           <number>10</number>
          </property>
          <item row="1" column="1">
           <widget class="QComboBox" name="cb_baud_rate">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>9</number>
            </property>
            <item>
             <property name="text">
              <string>1200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>9600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>28800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>38400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>57600</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>115200</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cb_device">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>波特率:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btn_refresh">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;
text-align:center;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="resource.qrc">
              <normaloff>:/icon/res/refresh.png</normaloff>:/icon/res/refresh.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2">
           <widget class="QPushButton" name="btn_connect">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>连接设备</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>设备:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="btn_baud_set">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>22</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="3">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" rowspan="5">
          <widget class="QTextEdit" name="recv_textEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="cb_hexDisplay">
           <property name="text">
            <string>HEX显示</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="cb_time">
           <property name="text">
            <string>加时间戳</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>编码:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cb_encode">
             <item>
              <property name="text">
               <string>UTF-8</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>GBK</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="btn_clearRecv">
           <property name="minimumSize">
            <size>
             <width>102</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>清空接收</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item row="0" column="0" rowspan="5">
          <widget class="QTextEdit" name="send_textEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="cb_hexSend">
           <property name="text">
            <string>HEX发送</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="cb_CRLF">
           <property name="text">
            <string>加回车换行</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="btn_clearSend">
           <property name="minimumSize">
            <size>
             <width>117</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>117</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>清空发送</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="btn_send">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
